eng/pipelines/templates/steps/analyze.yml

```diff
@@ -30,6 +30,7 @@ steps:
 
       Copy-Item "$(Build.ArtifactStagingDirectory)/PackageInfo" -Destination $directoryToPublish -Recurse
     displayName: 'Create Go APIView Artifact'
+    condition: succeeded()
```

Comment:
Should this condition match the step above this and add the IsSdkLibrary step? If not then there may not be any PackageInfo files in those cases.